uint8 thumb
uint8 index
uint8 long
uint8 ring
uint8 small